adventures/bucketlist/serializers.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.auth.models import User
+from rest_framework import serializers
+
+from bucketlist.models import Bucketlist, Item
+
+
+class ItemSerializer(serializers.ModelSerializer):
+    """Bucketlist Item Serializer"""
+    def validate_name(self, name):
```

Comment:
I decided to do that so that because it was giving me the django IntegrityError yet i wanted a customized error message.